src/Latte/Engine.php

```diff
@@ -294,6 +294,16 @@ public function addProvider($name, $value)
 
 
 	/**
+	 * Returns all providers.
+	 * @return string[]
```

Comment:
Ok I saw this above. What is difference between `string[]` and `array`? `string[]` is only array of strings without keys?